tests/functional/cleanup

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+. ~/test-projects/test1-openrc
```

Comment:
Consider making the path to the rc file a variable (maybe with a default value), rather than requiring a specific file structure in the home directory. Something like:

```
. ${ESI_TEST_CREDENTIALS:-./test-openrc}
```

This would default to a file named `test-openrc` in the current directory if `$ESI_TEST_CREDENTIALS` is unset.

You may also want to have a sane error message if the credentials file is missing.